uber/templates/macros.html

```diff
@@ -119,50 +119,51 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, collect_full_address=False, copy_address=False, name_prefix="") -%}
```

Comment:
I tweaked this macro slightly. I'll try to explain each change.

I renamed `is_prereg_dealer` to `is_dealer` because we also care if it's a dealer in the admin area in addition to the preregistration pages.